src/app.py

```diff
@@ -56,11 +61,8 @@ def stress_data(garth_client: Client) -> dict:
     Returns:
         dict: The daily stress data for the current date for the user.
     """
-    try:
-        return garth_client.connectapi(
-            f"/wellness-service/wellness/dailyStress/{datetime.date.today().isoformat()}")
-    except Exception as e:
-        print(e)
```

Comment:
Here you're handling all possible exceptions, printing the entire exception and returning `None`.